format.go

```diff
@@ -160,3 +162,57 @@ func (w *protobufWrapper) Encode(v interface{}) error {
 func (*protobufWrapper) Close() error {
 	return nil
 }
+
+// --------------------------------------------------------------------
+
+// ParquetFormat provides a Format implemention for Protobuf.
+var ParquetFormat = parquetFormat{}
+
+type parquetFormat struct{}
+
+// NewDecoder implements Format.
+func (parquetFormat) NewDecoder(r io.Reader) (FormatDecoder, error) {
+	rs, ok := r.(io.ReadSeeker)
```

Comment:
interesting, I am not sure if this is a bug or a feature, but it looks like we cannot read streams only files (as e.g. a bfs.Object is not seekable, neither is a compressed file)